model/flow/ledger_test.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // this benchmark can run with this command:
-//  go test -run=String -bench=.
+//  go test -run=Name -bench=.
```

Comment:
?

Reply:
[Fixed]Goland change String and Type in a bunch of unrelated places when I renamed the fields on the MsgAuthConfig